trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.ts

```diff
@@ -15,9 +16,11 @@ export const getUtilisationReportReconciliationByReportId = async (req: Request,
   const { reportId } = req.params;
 
   try {
-    const { addPaymentErrorSummary, isCheckboxChecked } = getAndClearAddPaymentFieldsFromRedirectSessionData(req);
+    const { addPaymentError, isCheckboxChecked } = getAndClearAddPaymentFieldsFromRedirectSessionData(req);
```

Comment:
Have we tested what happens if the user applies a filter, selects some fee records with differing payment currencies and or statuses and then clicks add a payment? Interested to know if the filter query params get persisted or not.